website/source/docs/provisioners/puppet-masterless.html.markdown

```diff
@@ -40,9 +40,10 @@ The reference of available configuration options is listed below.
 
 Required parameters:
 
-* `manifest_file` (string) - The manifest file for Puppet to use in order
-  to compile and run a catalog. This file must exist on your local system
-  and will be uploaded to the remote machine.
+* `manifest_file` (string) - This is either a path to a puppet manifest (`.pp`
+  file) _or_ a directory containing multiple manifests that puppet will apply.
```

Comment:
Is it worth noting the evaluation order, when there are multiple manifests? (i.e evaluated in alphabetical order)